www/include/views/graphs/exportData/ExportCSVServiceData.php

```diff
@@ -33,12 +33,7 @@
  *
  */
 
-function get_error($str)
```

Comment:
Why remove this function?

Reply:
@callapa When i included the bootstrap file, it showed an error that the function "get_error" is already defined.